lib/UI/UI.rb

```diff
@@ -54,9 +63,12 @@ def announce message
   end
 
   def announce_results(board, players)
-    announce("It's a tie") if board.tie?
```

Comment:
The `announce` method looks just like the `print_error_message` one. Maybe have a common one called `print`?